impl/src/bin/cargo-raze.rs

```diff
@@ -76,16 +80,21 @@ fn main() -> Result<()> {
     .and_then(|d| d.deserialize())
     .unwrap_or_else(|e| e.exit());
 
+  // Load settings
   let settings = load_settings("Cargo.toml")?;
   if options.flag_verbose.unwrap_or(false) {
     println!("Loaded override settings: {:#?}", settings);
   }
 
-  let mut metadata_fetcher: Box<dyn MetadataFetcher> = match options.flag_cargo_bin_path {
-    Some(ref p) => Box::new(CargoMetadataFetcher::new(p, /*use_tempdir: */ true)),
+  // Fetch metadata
+  let metadata_fetcher: Box<dyn MetadataFetcher> = match options.flag_cargo_bin_path {
+    Some(ref cargo_bin_path) => Box::new(CargoMetadataFetcher::new(
+      cargo_bin_path,
+      Url::parse(&settings.registry)?,
+      Url::parse(&settings.index_url)?,
+    )),
     None => Box::new(CargoMetadataFetcher::default()),
   };
-  let mut planner = BuildPlannerImpl::new(&mut *metadata_fetcher);
 
```

Comment:
Still a bunch of whitespace here between L89 "fetch metadata" and L118 "do planning".

Reply:
Sorry, I thought removing the comment form the earlier conversation meant this was not necessary. Updated